crates/relayer/src/event/monitor.rs

```diff
@@ -156,6 +156,10 @@ pub mod queries {
     pub fn ibc_channel() -> Query {
         Query::eq("message.module", "ibc_channel")
     }
+
+    pub fn ibc_query() -> Query {
```

Comment:
This should be added to the list of events to query, https://github.com/informalsystems/hermes/blob/master/crates/relayer/src/event/monitor.rs#L134-L141.

Reply:
https://github.com/informalsystems/hermes/pull/2915/commits/2be623da2cb287692330c51a5bb6562b4ef570d7